src/components/ImageScroller.js

```diff
@@ -147,14 +157,14 @@ export default function ImageScroller(props) {
                 height={192}
                 className={classes.image}
                 fixed
-                rotation={rotation}
+                rotation={rotations[idx] ? rotations[idx] : imageRotation}
```

Comment:
I'd suggest reverting back to the single value, but only applying it if this is the selected image.
When the user selects a new image, just reset the rotation to zero.